### PR TITLE
Fix IE11 Bug with donate header logo alignment

### PR DESCRIPTION
**Pull request by deniselashlley**

### CODE REVIEW

File: sass/themes/donate/donate-cr/components/header/_header.scss [hunk 1]
```diff
@@ -19,8 +19,7 @@ header[role=banner] {
 }
 
 .block-branding {
-  margin: 0 auto;
-  
+
```
Comment (AndyEPhipps) [inline sass/themes/donate/donate-cr/components/header/_header.scss]: umm

Comment (AndyEPhipps) [reply sass/themes/donate/donate-cr/components/header/_header.scss]: Do we not need more, uh, code here?

Comment (deniselashlley) [reply sass/themes/donate/donate-cr/components/header/_header.scss]: nope it works fine without in all browsers had it centred twice obviously IE11 didn't like it

Comment (AndyEPhipps) [reply sass/themes/donate/donate-cr/components/header/_header.scss]: For some reason, I thought this PR was just removing a space and nothing else; my bad!

**Pull request merged**